cheroot/workers/threadpool.py

```diff
@@ -157,6 +157,11 @@ def __init__(
             accepted_queue_timeout (int): timeout for putting request
                 into queue
         """
+        if min < 1:
+            raise ValueError('min must be > 0')
```

Comment:
It'd be nice to add tests for the invalid inputs too. Could even publish all the input validation additions as a separate PR, optionally.

Reply:
tests added